services/app-api/email/formatPackageDetails.js

```diff
@@ -15,6 +15,7 @@ export const formatPackageDetails = (data, config) => {
             ? `<br><b>Withdrawn By</b>: ${data.withdrawnByName}<br><b>Email Address</b>: ${data.withdrawnByEmail}`
             : `<br><b>Name</b>: ${data.submitterName}<br><b>Email Address</b>: ${data.submitterEmail}`
         }
+        ${data.title ? `<br><b>Waiver Title</b>: ${data.title}` : ""}
```

Comment:
I think it is the Amendment title, not the Waiver title?

Reply:
yes you are right! Thanks!